transactions/privateForwarder/create_private_forwarder.cdc

```diff
@@ -0,0 +1,26 @@
+import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import ExampleToken from 0xTOKENADDRESS
+import PrivateReceiverForwarder from 0xPRIVATEFORWARDINGADDRESS
+
+// This transaction creates a new private receiver in an account that 
```

Comment:
I'm not 100% sure but I think dapper account creation also removes the balance capability. Should we account for that here?

Reply:
Don't we need the balance capability to check storage capacity?

Reply:
Yes, so it would need to be added back to dapper accounts if they are removing it.

Reply:
oh I see, I thought you were saying we should remove it. I'll add a line to add it back

Reply:
Actually, the transaction that we would be using is `setup_and_create_forwarder.cdc` so we don't need it in this one. This one is just to create one on a normal account